src/ipahealthcheck/core/constants.py

```diff
@@ -6,27 +6,43 @@
 
 # Error reporting result
 SUCCESS = 0
-CRITICAL = 1
-ERROR = 2
-WARNING = 4
+WARNING = 10
+ERROR = 20
+CRITICAL = 30
 
 _levelToName = {
     SUCCESS: 'SUCCESS',
-    CRITICAL: 'CRITICAL',
-    ERROR: 'ERROR',
     WARNING: 'WARNING',
+    ERROR: 'ERROR',
+    CRITICAL: 'CRITICAL',
+}
+
+_nameToLevel = {
+    'SUCCESS': SUCCESS,
+    'WARNING': WARNING,
+    'ERROR': ERROR,
+    'CRITICAL': CRITICAL,
 }
 
 
 def getLevelName(level):
     """
-    Translate a level constant to a textual level name.
+    Translate between level constants and their textual mappings.
+
+    If the level is one of the predefined levels then returns the
+    corresponding string.
+
+    If a numeric value corresponding to one of the defined levels
+    is passed in instead the corresponding string representation is
+    returned.
     """
     name = _levelToName.get(level)
     if name is not None:
         return name
-    else:
-        return level
+    name = _nameToLevel.get(level)
+    if name is not None:
+        return name
+    return level
```

Comment:
If you want to keep the numeric and string constants separate (which is fine) you can simplify the code to:

return _levelToName.get(level) or _nameToLevel.get(level) or level

Reply:
Funny, I had looked at logging. Too much copy&paste.